core/src/main/resources/error/error-classes.json

```
@@ -1257,6 +1257,11 @@
           "AES-<mode> with the padding <padding> by the <functionName> function."
         ]
       },
+      "ANALYZE_TABLE_VIEW_NOT_CACHED" : {
+        "message" : [
+          "Temporary view <tableName> is not cached for analyzing columns."
```

Could you improve the error message by following the common structure as in other errors in `error-classes.json`:
1. State the issue
2. Give a suggestion how to solve it.

The error could be:
```suggestion
          "The ANALYZE TABLE FOR COLUMNS command can operate on temporary views that have been cached already. Consider to cache the view <view Name>."
```